### DISCUSSION TITLE
x509: certificate is valid for jaeger-operator-webhook-service.observability.svc, jaeger-operator-webhook-service.observability.svc.cluster.local, not jaeger-operator-webhook-service.devops.svc

### DISCUSSION THREAD

**Discussion by kingvip** [jaegertracing/jaeger#4055] [Q&A]
kubectl apply -f simplest.yaml

Error from server (InternalError): error when creating "allinone-jaeger-instance.yaml": Internal error occurred: failed calling webhook "mjaeger.kb.io": failed to call webhook: Post "https://jaeger-operator-webhook-service.devops.svc:443/mutate-jaegertracing-io-v1-jaeger?timeout=10s": x509: certificate is valid for jaeger-operator-webhook-service.observability.svc, jaeger-operator-webhook-service.observability.svc.cluster.local, not jaeger-operator-webhook-service.devops.svc

**Comment by timfpark** [comment]
I'm seeing the same thing - is it not possible to install the operator in a namespace other than "observability"?
